test/lib/server/services/eosReport/EosReportService.test.js

```diff
@@ -327,7 +327,8 @@ module.exports = () => {
 
     it ('should throw an error if no previous EoS report is found for autofilling', async () => {
         const response = await shiftService.getShiftData({ userId: 1 }, ShiftTypes.SLIMOS);
-        expect(response.infoFromPreviousShifter.errorMessage).to.equal('No SLIMOS EoS reports found from the previous shift');
+        const infoFromPreviousShifter = await response.infoFromPreviousShifter;
```

Comment:
Why do you have an await here?

Reply:
I await here because getInfoFromPreviousShifter() in shiftService returns a promise

Reply:
You should not return a promise as object property, you should await for it to resolve before return.